test/integration/name_conflict.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+	swarm_manage_cleanup
+	stop_docker
+}
+
+@test "name conflict short ID" {
+	start_docker 1
```

Comment:
`start_docker_with_busybox` as you don't want to pull `busybox` when you do the `create`